app/_src/kubernetes-ingress-controller/deployment/minikube.md

```diff
@@ -31,23 +31,55 @@ Kong is accessible. This is used to send requests into the
 Kubernetes cluster.
 
 1. Get the IP address at which you can access {{site.base_gateway}}:
-   
+   {% capture the_code %}   
+{% navtabs codeblock %}
+{% navtab kubectl %}
+```bash
+$ minikube service -n kong kong-proxy --url | head -1
+```
+{% endnavtab %}
+{% navtab Helm Charts %}
    ```bash
-   $ minikube service -n kong kong-proxy --url | head -1
-   # If installed by helm, service name would be "<release-name>-kong-proxy".
-   # minikube service <release-name>-kong-proxy --url | head -1
+   HOST=$(kubectl get svc --namespace kong kong-gateway-proxy -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   PORT=$(kubectl get svc --namespace kong kong-gateway-proxy -o jsonpath='{.spec.ports[0].port}')
+   echo $HOST:$PORT
    ```
-   The results should look like this:
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+{{ the_code | indent }}
+    The results should look like this:
+   {% capture the_code %}  
+{% navtabs codeblock %}
+{% navtab kubectl %}
+```bash
+http://192.168.99.100:32728
+```
+{% endnavtab %}
+{% navtab Helm Charts %}
    ```bash
```

Comment:
This should not be indented as we use the `indent` filter later.

<img width="476" alt="Screenshot 2023-10-05 at 11 46 43" src="https://github.com/Kong/docs.konghq.com/assets/59130/edf15c3b-fe65-45a8-b4dc-57686867dd82">

Reply:
addressed